.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    commit-message:
+      prefix: "fix(deps): "
```

```suggestion
    commit-message:
      prefix: fix
```
You can cut this down as dependabot will add the scope and trailing colon itself.
As an example: https://github.com/fastify/fastify/pull/3824